src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx

```diff
@@ -3,7 +3,8 @@ import React, { useState, useEffect } from 'react';
 import { hi } from 'date-fns/locale';
 import { useHistory } from 'react-router-dom';
 import FormLayout from '../new-appointment/components/FormLayout';
-import mockData from '../services/mocks/wellHive/providerServices.json';
+import ProviderAlert from './components/ProviderAlert';
+// import mockData from '../services/mocks/wellHive/providerServices.json';
```

Comment:
nit: remove commented import if it's not needed.

Reply:
we are eventually going to integrate with mockData, that's why I've left this import here